docker/nightly/centos7/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest code of OMI."
+
+ARG fork=Microsoft
+ARG branch=master
+ARG rootpassword=newpassword
+
+SHELL ["/bin/bash", "-c"] 
+RUN echo $rootpassword |passwd root --stdin
+RUN yum update -y; \
+    yum install -y git bind-utils gcc-c++ rpm-devel pam-devel openssl-devel rpm-build krb5-devel redhat-lsb-core;
+RUN git clone https://github.com/$fork/omi.git -b $branch; \
+    cd omi; \
+    pushd Unix; \
+    ./configure; \
+    make -j; \
+    make install; \
+    popd;
+EXPOSE 5985 5986
```

Comment:
I'm not sure what the default `omiserver.conf` file exposes when installed in that way (without trying it). Are both of these ports exposed? Do you want them both exposed?